node/src/commands/FlushMode.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+/**
+ * Defines flushing mode for {@link GlideClient.flushall|flushall} and {@link GlideClient.flushdb|flushdb} commands.
+ *
+ * See https://valkey.io/commands/flushall/ and https://valkey.io/commands/flushdb/ for details.
+ */
+export enum FlushMode {
+    /**
+     * Flushes synchronously.
+     *
+     * since Valkey 6.2 and above.
```

Comment:
nit: not sure how strict we want to be about this, but most of the "since" sections in the existing docs look a bit different. Do we want to align here or doesn't really matter?
```suggestion
     * since Valkey version 6.2.0.
```